app-backup/amanda/amanda-3.4.1.ebuild

```diff
@@ -0,0 +1,492 @@
+<<<<<<< HEAD
```

Comment:
You've got merge conflict markers committed.

This probably means you had a collision during a rebase/merge, and failed to run `git mergetool`, or that you ran a mergetool which didn't work, or you just did "git add  <file>" instead of handling it properly. 

This basically means the ebuild in tree at this point is broken and needs fixing to even work :)